Syntaxes/RSpec.tmLanguage

```diff
@@ -92,7 +94,7 @@
 		<key>behaviour</key>
 		<dict>
 			<key>begin</key>
-			<string>^\s*([fx]?(?:describe|context|feature))\b(?=.*\bdo\s*$)</string>
+			<string>^\s*([fx]?(?:RSpec\.describe|describe|context|feature))\b(?=.*\bdo\s*$)</string>
```

Comment:
I think you fixed it for one of the other two cases, but here it is still unfixed :)